.github/workflows/release-m1.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         id: vars
         run: |
           # Configure repo based on pushed tag if set
-          if [ -z "${GITHUB_REF}" ]; then
+          if [ ${{ github.event_name }} = 'push' ]; then
```

Comment:
the docs are a little bit confusing for this stuff (to me, at least)--does event_name = 'push' actually guarantee that GITHUB_REF is set?

https://docs.github.com/en/actions/learn-github-actions/variables says `This is only set if a branch or tag is available for the event type.`

Reply:
The docs are so shitty 🥲 Turns out GITHUB_REF is set no matter what. I tested by both pushing some tags and manually starting the workflow and it was always set

Reply:
fun fun. up to you based on your confidence level in that guarantee, but you could always check both.

Reply:
Want to make sure the workflow is fixed before I go on vacation so we can add that in the future if we need it